packages/shared/src/libs/helpers/get-start-of-day/get-start-of-day.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { startOfDay } from "date-fns";
+
+const getStartOfDay = (date: Date | number | string): Date => {
```

Comment:
we need to reduce variations
```suggestion
const getStartOfDay = (date: Date): Date => {
```